backend/src/main/java/gov/cdc/usds/simplereport/config/FeatureFlagsConfig.java

```diff
@@ -0,0 +1,43 @@
+package gov.cdc.usds.simplereport.config;
+
+import gov.cdc.usds.simplereport.db.model.FeatureFlag;
+import gov.cdc.usds.simplereport.db.repository.FeatureFlagRepository;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "features")
+@EnableScheduling
+@Data
+public class FeatureFlagsConfig {
+  @Autowired
+  @Getter(AccessLevel.NONE)
```

Comment:
Removing `@data` and switching to specific getters and setters would also eliminate the need to use the access levels here I believe.